openjdk/ProblemList_openjdk11-openj9.txt

```diff
@@ -158,6 +158,7 @@ java/math/BigInteger/PrimeTest.java https://github.com/AdoptOpenJDK/openjdk-test
 
 # jdk_net
 
+java/net/CookieHandler/B6791927.java    https://github.com/AdoptOpenJDK/openjdk-tests/issues/1436   generic-all
```

Comment:
We should reference the JBS issue and close 1436 as duplicate (https://bugs.openjdk.java.net/browse/JDK-8234007).  (same for all references in other files for this exclusion)